### PR TITLE
add device/group information to error log about some errors

### PR DESCRIPTION
**Pull request by AlvaroVega**
- EntityGenericError 
- BadTimestamp
- TypeNotFound
- DeviceNotFound
- MissingAttributes
- DuplicateDeviceId 
- DuplicateGroup 
- CommandNotFound
- GroupNotFound
- BadGeocoordinates

### CODE REVIEW

**Comment by AlvaroVega** [comment]
related: https://github.com/telefonicaid/iotagent-node-lib/pull/1536

**Comment by AlvaroVega** [comment]
curl -i -X POST 'http://localhost:7897/iot/json?i=dispNew&k=izc9cokegoy7kyfgdd6etipt6&t=null' -d '{ "c": 3, "d": 4}' -H 'content-type: application/json'
HTTP/1.1 404 Not Found
X-Powered-By: Express
Content-Type: application/json; charset=utf-8
Content-Length: 192
ETag: W/"c0-DBbtOjiLvN12/nt6uK1KmY6nLSw"
Date: Thu, 22 Feb 2024 07:08:33 GMT
Connection: keep-alive
Keep-Alive: timeout=5

{"name":"DEVICE_GROUP_NOT_FOUND","message":"Couldn\t find device group for fields: [\"resource\",\"apikey\"] and values: {\"resource\":\"/iot/json\",\"apikey\":\"izc9cokegoy7kyfgdd6etipt6\"}"}

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Add: log and return device/group information when EntityGenericError, TypeNotFound, DeviceNotFound, BadTimestamp, BadGeocoordinates, CommandNotFound, GroupNotFound, MissingAttributes, DuplicateDeviceId and DuplicateGroup errors
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Add: log and return device/group information when EntityGenericError, TypeNotFound, DeviceNotFound, BadTimestamp, BadGeocoordinates, CommandNotFound, GroupNotFound, MissingAttributes, DuplicateDeviceId and DuplicateGroup errors (iotagent-json#815)
```

File: test/unit/ngsi-ld/ngsiService/active-devices-test.js [hunk 2]
```diff
@@ -600,7 +600,7 @@ describe('NGSI-LD - Active attributes test', function () {
                 should.exist(error.name);
                 error.code.should.equal(207);
                 error.details.notUpdated.should.equal('someEntities');
-                error.message.should.equal('Error accesing entity data for device: light1 of type: Light');
+                //error.message.should.equal('Error accesing entity data for device: light1 of type: Light');
```
Comment (fgalan) [inline test/unit/ngsi-ld/ngsiService/active-devices-test.js:603]: Should this line be "recovered" ?

```suggestion
                error.message.should.equal('Error accesing entity data for device: light1 of type: Light');
```

Comment (AlvaroVega) [reply test/unit/ngsi-ld/ngsiService/active-devices-test.js:603]: Not, current message is more large and include a lot of fields of device.

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**